chapters/ch44.md

```diff
@@ -1,85 +1,71 @@
 
 > Case 1: Score​<sub>A​</sub>(​*S*<sup>x</sup>) > Score​<sub>A​</sub>(​*S*<sub>out</sub>)
 
---> _replace THIS LINE by your translation for the above line_
-
+Trường hợp 1: Điểm ​<sub>A​</sub>(​*S*<sup>x</sup>) > Điểm ​<sub>A​</sub>(​*S*<sub>out</sub>)
 
 > In this case, your learning algorithm has correctly given S<sup>x</sup> a higher score than S​<sub>out</sub>.​ Nevertheless, our approximate search algorithm chose S​<sub>out</sub> rather than S<sup>x</sup>. This tells you that out ​your approximate search algorithm is failing to choose the value of S that maximizes Score​<sub>A​</sub>(​*S*). In this case, the Optimization Verification test tells you that you have a search algorithm problem and should focus on that. For example, you could try increasing the beam width of beam search.
 
---> _replace THIS LINE by your translation for the above line_
-
+Trong trường hợp này, thuật toán học tập của bạn đã đưa S<sup>x</sup> một điểm chính xác cao hơn S​<sub>out</sub>.​ Tuy nhiên, thuật toán tìm kiếm gần đúng của chúng tôi đã chọn S​<sub>out</sub> thay vì S<sup>x</sup>. Điều này cho bạn biết rằng thuật toán tìm kiếm gần đúng của bạn không chọn giá trị S tối đa hóa Điểm​<sub>A​</sub>(​*S*). Trong trường hợp này, kiểm tra xác minh tối ưu hóa cho bạn biết rằng bạn có vấn đề về thuật toán tìm kiếm và nên tập trung vào đó. Ví dụ: bạn có thể thử tăng độ rộng chùm tia của tìm kiếm chùm tia.
 
 > Case 2: Score​<sub>A​</sub>(​*S*<sup>x</sup>) ≤ Score​<sub>A​</sub>(​*S*<sub>out</sub>)
 
---> _replace THIS LINE by your translation for the above line_
-
+Trường hợp 2: Điểm ​<sub>A​</sub>(​*S*<sup>x</sup>) ≤ Điểm ​<sub>A​</sub>(​*S*<sub>out</sub>)
 
 > In this case, you know that the way you’re computing Score​<sub>A​</sub>(.) is at fault: It is failing to give a strictly higher score to the correct output ​*S*<sup>x</sup> than the incorrect ​S​<sub>out</sub>. The Optimization Verification test tells you that you have an objective (scoring) function problem. Thus, you should focus on improving how you learn or approximate Score​<sub>A​</sub>(​*S*) for different sentences ​*S​*.
 
---> _replace THIS LINE by your translation for the above line_
-
+Trong trường hợp này, bạn biết cách bạn tính toán Điểm​<sub>A​</sub>(.) có lỗi: Không thể cho điểm cao hơn hoàn toàn cho đầu ra chính xác ​*S*<sup>x</sup> so với ​S​<sub>out</sub> không chính xác. Kiểm tra xác minh tối ưu hóa cho bạn biết rằng bạn có vấn đề về hàm tính điểm mục tiêu. Vì vậy, bạn nên tập trung vào việc cải thiện cách bạn học hoặc tính gần đúng Điểm​<sub>A​</sub>(​*S*) cho các câu khác nhau ​*S​*.
```

Comment:
* [ ] "bạn biết ~~cách bạn~~ _việc_ tính toán"
* [ ] "strictly" -> "hẳn"
* [ ] "bạn có vấn đề về hàm ~~tính điểm~~ mục tiêu _(tính điểm)_"
* [ ] "approximate" = "xấp xỉ". Để thuận tai, đoạn này có thể dịch là:
> Vì vậy, bạn nên tập trung vào việc cải thiện ~~cách bạn~~ _thuật toán_ học hoặc ~~tính gần đúng~~ _cách xấp xỉ_ Điểm​<sub>A​</sub>(​*S*) cho các câu khác nhau ​*S​*.